cfgov/jinja2/v1/events/_macros.html

```diff
@@ -133,12 +133,11 @@
         <div class="event-venue_details">
             <header class="event-venue_header">
                 {% if ( event_state == 'present' ) %}
-                <a class="btn
-                          event-venue_live-btn"
-                   href="/">
+                <btn class="btn
+                          event-venue_live-btn">
                     <span class="cf-icon cf-icon-livestream"></span>
                     <span class="event-venu_live-btn_text">Live</span>
-                </a>
+                </btn>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/eee3eaacb98e5439b4700e260e20a6e59c14a9ab. Really Steven? `btn`